Automattic/wp-calypso
# Enable new categories in the category selector

#### Changes proposed in this Pull Request

* Enable the new categories for searching
* Add searching components from https://github.com/Automattic/wp-calypso/pull/62685

#### Testing instructions

* http://calypso.localhost:3000/plugins/
* Dropdown should now include more categories to browse
* Check /pages isn't broken p1650417714923679-slack-C01QG4Y91RR

#### Screenshot

![Screenshot 2022-04-19 at 19-42-49 Plugins ‹ Site Title — WordPress com](https://user-images.githubusercontent.com/811776/163976418-ce05e068-2dff-4e73-84e1-098cbec91504.png)

Related to https://github.com/Automattic/wp-calypso/issues/62264
